LoggingKata/TacoParser.cs

```diff
@@ -1,18 +1,62 @@
 ﻿namespace LoggingKata
+
+
 {
     /// <summary>
     /// Parses a POI file to locate all the Taco Bells
     /// </summary>
     public class TacoParser
     {
         readonly ILog logger = new TacoLogger();
-        
+
+
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
+            // Take your line and use line.Split(',') to split it up into an array of strings, separated by the char ','
+            var cells = line.Split(',');
+
+            // If your array.Length is less than 3, something went wrong
+            if (cells.Length < 3)
+            {
+                return null;
+            }
+
+            // grab the latitude from your array at index 0
+            string latitude = cells[0];
+
+
+            // grab the longitude from your array at index 1
+            string longitude = cells[1];
+
+            // grab the name from your array at index 2
+            string name = cells[2];
+
+            // Your going to need to parse your string as a `double`
+            double lat = double.Parse(latitude);
+            double lon = double.Parse(longitude);
```

Comment:
These are liable to throw an exception, but the current test cases don't cover it. Consider making these parses more resilient using a TryParse